Makefile

```
@@ -4,11 +4,12 @@ BIN := $(VENV)/bin
 
 PYTHON := env PYTHONPATH=$(PYTHONPATH) $(BIN)/python
 PIP := $(BIN)/pip
+PYTEST := $(BIN)/pytest
 
 REQUIREMENTS := -r requirements.txt
 PRE_COMMIT := $(BIN)/pre-commit
 
-APP_NAME := knot-thing-simulator
+APP_NAME := knot_thing_simulator
```

I believe the [convention](https://docs.python.org/3.3/c-api/module.html?highlight=pymodule) is `PYMODULE`, but is really up to you :stuck_out_tongue: 

```suggestion
PYMODULE := knot_thing_simulator
```

---

I believe that this change should be a commit that is only related with moving the files to the "knot_sim" folder and adapting the files to support this change.

---

Done (both - created a pymodule var to held test coverage report and another to have the main app module)